### PR TITLE
Fix 5776 - `make` gives `ocamlfind: No such file or directory` on every execution

### PR DESCRIPTION
**Pull request by maximedenes**

### CODE REVIEW

File: tools/CoqMakefile.in [hunk 1]
```diff
@@ -189,7 +187,12 @@ else
 CAMLP4EXTEND=
 endif
 
+ifneq (, $(shell which "$(OCAMLFIND)" 2> /dev/null))
```
Comment (JasonGross) [inline tools/CoqMakefile.in]: Does this work, or does the leading space mess things up? (What does make do in `ifneq (, )`?)

Comment (maximedenes) [reply tools/CoqMakefile.in]: Good question, let's remove the space!

Comment (maximedenes) [reply tools/CoqMakefile.in]: Btw, if you could test the patch on the setup where the message was showing up, and confirm it is gone, it would be ideal.

**Comment by maximedenes** [comment]
The patch was probably wrong, but I've updated it. Thanks, @JasonGross. I'd feel better if you can confirm it works for you, though.

**Comment by JasonGross** [comment]
I'll aim to take a look at it later today.  Might be a bit tricky, though, since this is source and I got the issue from the prebuilt binary.  Any chance you could get me a Windows installer for this PR?

**Comment by Zimmi48** [comment]
AppVeyor creates binaries automatically. You'll just have to wait for it to run.

**Comment by maximedenes** [comment]
Ok, this fix makes the whole test suite fail, since `coqtop -config` emits a Windows path for `ocamlfind`, not a Cygwin path. I'm a bit puzzled as to what is the scenario for using `coq_makefile` on windows. Is it meant to be used inside our outside cygwin? @gares ?

**Pull request merged**